engine/battle_anims/bg_effects.asm

```diff
@@ -2567,8 +2567,8 @@ BGEffects_LoadBGPal0_OBPal1:
 	ld b, a
 	ld c, $1
 	call CopyPals
-	ld hl, wOBPals2 palette 1
-	ld de, wOBPals1 palette 1
+	ld hl, wOBPals2 palette PAL_BATTLE_OB_PLAYER
```

Comment:
`ld hl, wBGPals2` and `ld de, wBGPals1` above should have `palette PAL_BATTLE_BG_PLAYER`, and the whole function should be renamed to `BGEffects_LoadPlayerPals`.